data/uhc_pack/functions/pre_generation/stage/step/marker/origin/check_loaded.mcfunction

```diff
@@ -0,0 +1,3 @@
+execute if score Dimension uhcPG matches -1 in minecraft:the_nether store success score AlreadyLoadedOrigin uhcPG run forceload query ~ ~
+execute if score Dimension uhcPG matches 0 in minecraft:overworld store success score AlreadyLoadedOrigin uhcPG run forceload query ~ ~
+execute if score Dimension uhcPG matches 1 in minecraft:the_end store success score AlreadyLoadedOrigin uhcPG run forceload query ~ ~
```

Comment:
Remove this line please

Reply:
These functions are needed sadly, or we'll have to find a neater way to implement them.

Spawning the armor stands in the overworld (at the lobby), then teleporting them to the dimension they are needed in, gave pretty unreliable results at best. I changed it to spawn at the coordinates of the lobby in the overworld, but then in the dimension itself. This was far more reliable, but does require the chunk where the armor stand spawns to be loaded (because of that armor stand requiring manual selection)

I could remove these functions, but that would either result in the origin chunk remaining loaded, or being unloaded regardless of whether it was supposed to or not.

Reply:
What I mean is literally remove the one line. At no point do you set Dimension = 1. We aren't pre-generating the end.

Reply:
Aaaah hahah, it was showing all three, that's what confused me 😄